templates/case-study/episensor.html

```diff
@@ -0,0 +1,267 @@
+      </p>
+      <p>
+
+        Brendan Carroll continues: “If there’s a problem, it’s just not an option to visit hundreds, thousands, or even millions of sites to ‘press the reset button’. Server operating systems are built on a philosophy that you’re always connected and always have physical access to the hardware – that’s just not the case for the solutions we deploy.”
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) image_wrapper() -%}
+    {%- if slot == 'image_content' %}
+      {{ image(url="https://assets.ubuntu.com/v1/6503bd56-image2.png",
+            alt="",
+            width="1848",
+            height="771",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container__image"}) | safe
+      }}
+    {%- endif -%}
+  {%- endcall -%}
+  {%- call(slot) text(title="Solution") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        In 2016, EpiSensor set out to find an operating system that was better suited to the Gateway use case. The company wanted an OS that its engineering team was familiar with, and that would meet the needs of its enterprise customers. EpiSensor considered building its own version of Debian running Docker, or using Yocto Linux or CentOS, but none of these offered the right balance of security, reliability, and remote management capability. Fortunately, EpiSensor discovered one OS that was an ideal match for its requirements – Ubuntu Core.
+      </p>
+      <p>
+        Ubuntu Core is a minimal and strictly confined version of Ubuntu built for deploying embedded devices, designed with security in mind. Delivered and maintained by Canonical, the OS receives up to 12 years of continuous security updates, and features full disk encryption, secure boot, and an array of additional security tools.
+      </p>
+      <p>
+        “Ubuntu Core represented the best solution we found for our IoT Gateway,” confirms Brendan Carroll. “A hardened and maintained environment to run our application software with fine-grained permissions that control access to features of the operating system and hardware peripherals, with robust delivery of automatic software updates that can be rolled back in the case of failure.” 
+      </p>
+      <p>
+        These automatic updates, as well as easy remote management of the device fleet as a whole, are made possible by the snap packaging format and EpiSensor’s Dedicated Snap Store. Snaps are applications bundled with their dependencies, and they can be fully containerised and immutable for added security. The Dedicated Snap Store gives EpiSensor a tailored, centralized platform from which it can remotely publish, distribute, and manage its snaps – with reliable over-the-air (OTA) updates – in a way that is both automated and controlled.
```

Comment:
```suggestion
        These automatic updates, as well as easy remote management of the device fleet as a whole, are made possible by the snap packaging format and EpiSensor’s Dedicated Snap Store. Snaps are applications bundled with their dependencies, and they can be fully containerized and immutable for added security. The Dedicated Snap Store gives EpiSensor a tailored, centralized platform from which it can remotely publish, distribute, and manage its snaps – with reliable over-the-air (OTA) updates – in a way that is both automated and controlled.
```